financialaid/management/commands/generate_exchange_rates.py

```diff
@@ -0,0 +1,16 @@
+"""
+Generates CurrencyExchangeRate objects
+"""
+from django.core.management import BaseCommand
+
+from financialaid.tasks import sync_currency_exchange_rates
+
+
+class Command(BaseCommand):
+    """
+    Generates CurrencyExchangeRate objects by calling sync_currency_exchange_rates
+    """
+    help = "Generates CurrencyExchangeRate objects by calling sync_currency_exchange_rates"
+
+    def handle(self, *args, **kwargs):  # pylint: disable=unused-argument
+        sync_currency_exchange_rates()
```

Comment:
I just took a look at the implementation of the `sync_currency_exchange_rates()` function, and it doesn't handle API failures at all. This is a big problem -- again, consider that the person running this command might be an operations person who does not have intimate knowledge of the codebase. API failures happen, and they need to be accounted for.

Can you add a bit of failure handling logic to the `sync_currency_exchange_rates()` function, and make sure that it handles failures gracefully? At a minimum, the function should call [`raise_for_status()`](http://docs.python-requests.org/en/master/api/#requests.Response.raise_for_status) on the HTTP response _before_ calling `.json()` on it. This management command should use a `try`/`except` block when calling the function, and provide a helpful error message that includes the exact URL that failed, as well as the HTTP status code of the response.